crate/utils/Cargo.toml

```diff
@@ -20,6 +19,7 @@ serde_json = { workspace = true }
 strum = { workspace = true }
 strum_macros = { workspace = true }
 thiserror = { workspace = true }
+tiny-keccak = { version = "2.0.2", features = ["shake"] } # TODO(E.C): should be fixed in cosmian_crypto_core
```

Comment:
What is this todo? Why can't you use the `kmac!` from crypto_core?

Reply:
kmip_utils.rs uses the new macro `kdf256` but cannot build without tiny-keccak

Reply:
Ah... indeed... would this be a patch release in CryptoCore? If it is, then maybe we could patch this first?

Reply:
yes a patch release is a good idea

Reply:
creating this crypto_core issue: https://github.com/Cosmian/crypto_core/issues/50